examples/bench/example.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
The license must look exactly like https://github.com/gravitational/teleport/blob/master/constants.go#L1-L15 (with year 2020), including spacing.
Same for all `.go` files in this PR

Reply:
Should I replace Gravitational with Teleport?

Reply:
Nope, company legal name is still Gravitational